commands/weather.py

```diff
@@ -0,0 +1,31 @@
+async def weather(ctx, zipcode):
+    """ Gets the current weather for given US zip code """
+    url = 'https://api.openweathermap.org/data/2.5/weather?'
+    url += f"zip={zipcode}"
+    url += f"&appid={WEATHER_API_KEY}"
```

Comment:
It probably makes more sense to combine these lines, now that we are using f strings:
```suggestion
    url += f"zip={zipcode}&appid={WEATHER_API_KEY}"
```